cookbooks/aws-parallelcluster-slurm/files/default/head_node_slurm/slurm/config_utils.py

```diff
@@ -0,0 +1,98 @@
+import math
+from typing import List, Tuple
+
+def _get_instance_types(compute_resource_config) -> List[str]:
```

Comment:
Nitpick (maybe addressed later?): these are now functions used by external scripts/classes, so let's drop the leading underscore.

Reply:
Good catch.
I fixed ` get_template_folder()` but I forgot the other ones since I moved them in bulk.

Fixing.